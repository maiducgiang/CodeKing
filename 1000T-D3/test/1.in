4
9857 9181 2674 467
160 8015 9374 3948
1212 2310 2554 3548
9299 3408 1446 7965
