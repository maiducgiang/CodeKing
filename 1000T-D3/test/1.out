6095151
74264450
7104468
18374322
